packages/invoices/src/OutboundShipment/DetailView/modals/ItemDetailsModal.tsx

```diff
@@ -131,13 +133,19 @@ const usePackSizeController = (
     )
   );
 
-  const options = packSizes.map(packSize => ({
-    label: String(packSize),
-    value: packSize,
-  }));
+  const anySize = [];
+  if (packSizes.length > 1) {
+    anySize.push({ label: t('label.any'), value: -1 });
```

Comment:
popping in a random entry with a magic number as the value! 
it's bad, yes. it works though 😬